### ISSUE TITLE
Android Standalone App, Error only on first load

### ISSUE THREAD

**Issue by jbakerdev** [expo/expo#737]
![image](https://user-images.githubusercontent.com/8162030/31259929-95d27c6a-aa18-11e7-86ce-fd42bb8ab1ba.png)

On a given android device (tried s7 and 5X), this happens once and only once when the app is opened from the Play Store. If you reopen the app everything is fine. If you uninstall/reinstall the app, everything is fine. 

built APK using SDK 19

**Comment by jimmylee** [comment]
Would love to know more about your project, the dependencies you're using, etc.

**Comment by dikaiosune** [comment]
Hi! It seems like there might have been an error when including the bundle in your standalone app. Can you try building again, and seeing if that fixes it in the new APK?

**Comment by jbakerdev** [comment]
Updating to SDK 20 seems to have solved the issue.

**Comment by jbakerdev** [comment]
I was wrong. This issue still occurs on phones in the wild. Re-building the APK has no effect.

Dependencies:

"react-native-animatable": "^1.2.2",
    "react-native-aws3": "0.0.8",
    "react-native-circular-slider": "^1.0.1",
    "react-native-datepicker": "^1.6.0",
    "react-native-elevated-view": "0.0.4",
    "react-native-event-source": "^1.0.0",
    "react-native-html-to-text": "0.0.5",
    "react-native-modalbox": "^1.4.1",
    "react-native-orientation": "^2.1.0",
    "react-native-parallax-scroll-view": "^0.19.0",
    "react-native-parsed-text": "0.0.18",
    "react-native-pathjs-charts": "0.0.30",
    "react-native-progress-circle": "^1.0.3",
    "react-native-scrollable-tab-view": "^0.8.0",
    "react-native-simple-store": "^1.3.0",
    "react-native-slider": "^0.11.0",
    "react-native-swiper": "1.5.4",
    "react-native-timeago": "^0.3.0",
    "react-redux": "^5.0.5",
    "redux": "^3.6.0",
    "redux-thunk": "^2.2.0"

**Comment by monsterdream** [comment]
Hi,
I have similar problem which occurs once per installation, only at the beginning. I can see that for 3 months nobody solve this problem, or maybe?

![Image](https://cl.ly/2E0O26143i12/Image%202018-01-11%20at%201.32.30%20PM.png)

Together with dependencies:

**Comment by jesseruder** [comment]
@jbakerdev this should have been fixed in sdk 23. Are you still seeing it there?

@monsterdream that looks like a separate issue. Can you open a new issue or a forum post?

**Comment by jbakerdev** [comment]
This is no longer happening since upgrading to SDK 24

**Comment by jesseruder** [comment]
Thanks for letting us know @jbakerdev!

**Comment by abhishekbajpai01** [comment]
This happened today as well in SDK 28, after installing an app from play store.
And after clearing cache it works fine.
